ios/Helio.xcodeproj/xcshareddata/xcschemes/HelioDev.xcscheme

```diff
@@ -60,9 +60,8 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
-      shouldUseLaunchSchemeArgsEnv = "YES">
-      <Testables>
-      </Testables>
+      shouldUseLaunchSchemeArgsEnv = "YES"
+      shouldAutocreateTestPlan = "YES">
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `shouldAutocreateTestPlan` attribute has been added and set to "YES". This means Xcode will automatically create a test plan if none exists. Ensure this is the desired behavior.
```diff
- shouldUseLaunchSchemeArgsEnv = "YES">
+ shouldUseLaunchSchemeArgsEnv = "YES"
+ shouldAutocreateTestPlan = "YES">
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->